src/documents/views.py

```diff
@@ -445,6 +445,10 @@ def comments(self, request, pk=None):
                 )
                 c.save()
 
+                from documents import index
+
+                index.add_or_update_document(self.get_object())
```

Comment:
Just to make sure, is get_object going to get the same `doc` as gotten by the primary key above?

Reply:
I think so,
```
print(doc.pk, self.get_object().pk)
```
=
```
204 204
```

But ofc feel free to double-check me here

Reply:
Ok great